utils/gitlab_api.py

```diff
@@ -110,12 +110,26 @@ def check_group_exists(self, group_name):
             return False
         return True
 
+    def get_string_access_level(self, access):
```

Comment:
can you refactor the code that translates from access to number? I see this is repeated

Reply:
one of the times it is string -> number the other it is number -> string